md6/src/consts.rs

```diff
@@ -18,10 +17,9 @@ pub const MD6_C: usize = 16;
 /// Size of compression input block in words
 pub const MD6_N: usize = 89;
 
-// These five values give lengths of the components of compression
-// input block; they should sum to MD6_N.
-
-// Q words in a compression block (>= 0)
+/// These five values give lengths of the components of compression
+/// input block; they should sum to MD6_N.
```

Comment:
The docstring was needed only for "Q words in a compression block (>= 0)". These two lines should be kept as usual comments (i.e. as `//`) since they address the block of constants, not just `MD6_Q`.